data/src/main/java/smile/data/Attribute.java

```diff
@@ -73,30 +73,18 @@
      */
     private String description;
 
```

Comment:
Please reverts the changes to Attribute.java and NumericAttribute.java

Reply:
Done